src/applications/dhp-connected-devices/tests/components/TermsAndConditions.unit.spec.jsx

```diff
@@ -19,14 +19,17 @@ describe('Terms and Conditions', () => {
     expect(screen.getByTestId('fitbit-terms-and-conditions')).to.exist;
     expect(screen.getByTestId('fitbit-terms-and-conditions-content')).to.exist;
     expect(
-      screen.getByText('Sign into Fitbit and share your Fitbit data with VA'),
-    ).to.exist;
-    expect(screen.getByText('View Terms')).to.exist;
+      screen.getByTestId('fitbit-terms-and-conditions').textContent,
+    ).to.have.string('Sign into Fitbit and share your Fit');
```

Comment:
Should we use the whole text here? 
```suggestion
    ).to.have.string('Sign into Fitbit and share your Fitbit data with VA');
```